spec/system/index_spec.rb

```diff
@@ -4,10 +4,16 @@
 
 RSpec.describe 'Index', type: :system do
   describe 'index page' do
-    it 'contains smartflix' do
+    it 'contains Smartflix' do
```

Comment:
these are identical specs, they are just testing different text on the same page. You can group them together and add more `expect` logic.